cookbooks/bach_sqoop/attributes/default.rb

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright 2015, Bloomberg Finance L.P.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+default[:bach][:sqoop][:install_dir] = '/usr/lib/sqoop'
+
+default[:bach][:sqoop][:lib_dir] =
+  default[:bach][:sqoop][:install_dir] + '/lib'
+
+# Place JDBC jars in this hash.
+# They will be installed to the sqoop lib_dir.
+default[:bach][:sqoop][:jdbc_jars] =
```

Comment:
For JDBC specifically, I think they should all offer self-contained jars to make this easy.  (They're already putting up a much higher barrier than java developers expect by putting the jars behind a click-through license)

For something unrelated to JDBC, we would have to write POMs or cache an upstream artifact repository, then use maven to install the dependency tree.